src/styles/main.scss

```diff
@@ -1,7 +1,255 @@
+    & .logo {
+      height: 30px;
+      width: 30px;
+    }
+    & .logo-bar-nav {
+      display: flex;
+      justify-content: space-between;
+      & .logo-bar-image {
+        object-fit: cover;
+        height: 30px;
+        width: 30px;
+        border-radius: 50%;
+      }
+      & a {
+        text-decoration: none;
+        color: #3d4e61;
+        font-style: "Heavy";
+        font-size: 16px;
+        padding-right: 40px;
+      }
+    }
+  }
+  & .banner {
+    background-image: url(../images/main-photo.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 464px;
+  }
+}
+.main {
+  padding: 40px calc(50% - 451px) 25px;
+  display: flex;
+  justify-content: space-between;
+  & .apartment {
+    color: #3d4e61;
+    max-width: 492px;
+    & .sec-head {
+      font-size: 36px;
+      line-height: 42px;
+
+    }
+    & .sub-head {
+      font-size: 16px;
+      line-height: 19px;
+      color: #687c94;
+    }
+    & .host-button{
+      width: 150px;
+      height: 50px;
+      outline: none;
+      font-size: 16px;
+      color: #118488;
+      background-color: white;
+      border: 1px solid #118488;
+      box-sizing: border-box;
+      border-radius: 4px;
+    }
+    & .apart-details {
+      display: flex;
+      & p {
+        padding-right: 82px;
+        font-size: 16px;
+        line-height: 19px;
+        color: #3d4e61;
+      }
+    }
+    & .apart-info {
+      font-size: 16px;
+      line-height: 24px;
+      color: #3d4e61;
+    }
+    & .contact-host{
+      @include smallHeading;
+    }
+    & .amenity-head{
+      @include smallHeading;
+    }
+    & .facilities span{
+      margin-left: 120px;
+    }
+    & .show-all{
+      @include smallHeading;
+    }
+  }
+  & form{
+    height: 330px;
+    width: 319px;
+    padding: 30px 30px 38px;
+    box-sizing: border-box;
+    border: 1px solid #E1E7ED;
+    border-radius: 3px;
+
+    & .price{
+      font-size: 16px;
+      line-height: 22px;
+      color: #3D4E61;
+    }
+    & input{
+      @include formInput;
+      border-radius: 3px;
+      border: 1px solid #E1E7ED;
+      padding: 14px 17px;
+      margin-bottom: 16px;
+    }
+  }
+  & .form-button{
+     @include formInput;
+     border-radius: 4px;
+     background: #FD5C63;
+     border: 1px solid #FD5C63;
+     color: white;
+     margin-top: 8px;
+  }
+  & .review{
+    & .rev-block{
+      position: relative;
+      &::before{
+      content: "";
+      height: 50px;
+      width: 50px;
+      border-radius: 50%;
+      background-image: url(../images/headshot.jpg);
+      position: absolute;
+    }
+    & .name{
+      padding-left: 80px;
+      font-weight: bolder;
+    }
+    & .date{
+      padding-left: 80px;
+    }
+  }
+  }
+}
+
+.foot{
+  background: #FFFFFF;
+  box-shadow: 0px -1px 0px #E1E7ED;
+  border: 1px solid  #E1E7ED;
+  padding: 40px calc(50% - 451px) 25px;
+  display: flex;
+  & .foot-nav{
+    list-style: none;
+    padding: 0;
+    margin-right: 300px;
```

Comment:
better to use `space-between` or `space-around` on parent element instead of this margin